releasenotes/notes/asm-helm-chart-integration-4e6f5a4cf263592a.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
```

Comment:
This is a change that only affects the cluster agent, so the release note should be moved to `releasenotes-dca/notes`

Reply:
fixed in 675b22f098